### PR TITLE
simple configuration handling

### PR DESCRIPTION
**Pull request by Brianktoo**
add `configuration handling feature ` to handle configurations

### CODE REVIEW

File: configuration_handling/config.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+class TestingConfig(Config):
+    TESTING = True  
+     
+    DB_NAME = 'testing_db'
+    DB_USERNAE = 'root'
+    DB_PASSWORD = 'password'
+    SESSION_COOKIE_SECURE = False
+    
+     
+
```
Comment (codetaskmaster) [inline configuration_handling/config.py:33]: Whitespace!

File: configuration_handling/app.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+env=os.getenv("ENV")
+print(f'you are working on {env} environment')
+@app.route('/')
+def hello_world():
+    print(app.config['DB_NAME'])
+    
+    return 'Hello, World!'
+
+if __name__ == '__main__':
+    app.run(debug=True)
```
Comment (codetaskmaster) [inline configuration_handling/app.py:15]: Whitespace!

**Review by codetaskmaster — CHANGES_REQUESTED** [review]

File: configuration_handling/app.py (hunk 2 above)
Comment (Brianktoo) [reply configuration_handling/app.py]: done.  white space not with space

Comment (Brianktoo) [reply configuration_handling/app.py]: I have also added config.py to git ignore.  is it suppose to be that way?

File: configuration_handling/app.py [hunk 3]
```diff
@@ -12,4 +12,5 @@ def hello_world():
     return 'Hello, World!'
 
 if __name__ == '__main__':
-    app.run(debug=True)
\ No newline at end of file
+    app.run(debug=True)
+
```
Comment (codetaskmaster) [inline configuration_handling/app.py:16]: Check this sir!

**Review by codetaskmaster — CHANGES_REQUESTED** [review]

File: configuration_handling/app.py (hunk 2 above)
Comment (codetaskmaster) [reply configuration_handling/app.py]: > I have also added config.py to git ignore. is it suppose to be that way?

You should not. These are the setup settings for your app

**Pull request merged**